tests/test_app.py

```diff
@@ -4,7 +4,7 @@
 from webob import Response
 from webtest.compat import to_bytes
 from webtest.compat import PY3
-from webtest.compat import OrderedDict
+from collections import OrderedDict
```

Comment:
feel free to suggest where this modified import should reside.  i realize it probably shouldn't be in the middle of the webtest imports, but i couldn't discern an order to the imports and identify where it should go instead.